app/src/main/java/com/amaze/filemanager/services/ftpservice/FTPService.java

```diff
@@ -354,6 +360,12 @@ public static byte byteOfInt(int value, int which) {
         return (byte) (value >> shift);
     }
 
+    public static int getPort(Context context)
```

Comment:
You should pass a reference to SharedPreferences, instead of creating it, so that you don't accidentally have a method creating an instance when you already have one.